component/component.go

```diff
@@ -33,27 +33,27 @@ import (
 // is created, starts, runs and is shutdown again.
 type Component interface {
 	// Start tells the component to start. Host parameter can be used for communicating
-	// with the host after Start() has already returned. If error is returned by
+	// with the host after Start() has already returned. If an error is returned by
 	// Start() then the collector startup will be aborted.
 	// If this is an exporter component it may prepare for exporting
 	// by connecting to the endpoint.
 	//
 	// If the component needs to perform a long-running starting operation then it is recommended
 	// that Start() returns quickly and the long-running operation is performed in background.
 	// In that case make sure that the long-running operation does not use the context passed
-	// to Start() function since that context will be cancelled soon and can abort the long-running operation.
-	// Create a new context from the context.Background() for long-running operations.
+	// to Start() function since that context will be cancelled soon and can abort the long-running 
+	// operation. Create a new context from the context.Background() for long-running operations.
 	Start(ctx context.Context, host Host) error
 
-	// Shutdown is invoked during service shutdown. After Shutdown() is called, if the component accept data in
-	// any way, it should not accept it anymore.
+	// Shutdown is invoked during service shutdown. After Shutdown() is called, if the component 
+	// accepted data in any way, it should not accept it anymore.
 	//
-	// If there are any background operations running by the component they must be aborted as soon as possible.
-	// Remember that if you started any long-running background operation from the Start() method that operation
-	// must be also cancelled. If there are any buffer in the component, it should be cleared and the data sent
-	// immediately to the next component.
-	//
-	// Once the Shutdown() method returns the component's lifecycle is completed. No other
+	// If there are any background operations running by the component they must be aborted as 
+	// soon as possible. Remember that if you started any long-running background operations from 
+	// the Start() method those operations must be also cancelled. If there are any buffers in the 
+	// component, it should be cleared and the data sent immediately to the next component.
+	// 
+	// Once the Shutdown() method returns, the component's lifecycle is completed. No other
```

Comment:
```suggestion
	// Once the Shutdown() method returns the component's lifecycle is completed. No other
```

Don't think a comma is necessary here.  Alternately:

```suggestion
	// The component's lifecycle is completed once the Shutdown() method returns. No other
```

Reply:
Agreed that the second alternative sounds good.